SpywareFilter/sections/allowlist.txt

```diff
@@ -37,6 +37,8 @@
 !#endif
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/155484
+@@||srv.tunefindforfans.com/showads/track/srvclk.php?
```

Comment:
I assume this only happens with content blockers, so it's better to limit the rule for iOS and Safari extension.